resources/helm/overlays/istio_cni/templates/daemonset-v2.3.yaml

```diff
@@ -0,0 +1,128 @@
+{{ if .Values.cni.enabled }}
+{{- if and (.Values.cni.supportedReleases) (eq .Values.cni.instanceVersion "v2.3") }}
```

Comment:
Why check if `supportedReleases` is not nil? Shouldn't this check whether `supportedReleases` contains `v2.3`?

Reply:
I checked the versions code and I don't see a case that supportedReleases will be nil as long as ver.Info.MinimumSupportedVersion is not InvalidVersion
https://github.com/maistra/istio-operator/blob/maistra-2.3/pkg/controller/versions/versions.go#L85

and yes, I updated that line with 
{{- if and (has "v2.3" .Values.cni.supportedReleases) (eq .Values.cni.instanceVersion "v2.3") }}
If a future release operator will deprecate v2.3, the first if check will skip rendering that daemonset.